debian/rules

```diff
@@ -1,15 +1,9 @@
 #!/usr/bin/make -f
 
-include /usr/share/dpkg/pkg-info.mk
-
-export DH_VERBOSE=1
```

Comment:
In general I'm against setting DH_VERBOSE=1 for no specific reason.
If we always want debhelper to be more verbose then we should fix
its default verbosity level.